### ISSUE TITLE
Exa crashes on an unsertain condition

### ISSUE THREAD

**Issue by cesasol** [ogham/exa#738]
I just wanted to ls my wallpaper folder and exa crashed. I have some weird links and old broken symlinks so I will try to pinpoint the issue soon.

Here is the backtrace, using latest commit on master.

```
➜ RUST_BACKTRACE=full ls -la
thread 'thread 'thread 'thread 'thread 'thread '<unnamed><unnamed>thread '' panicked at '<unnamed>' panicked at '<unnamed>called `Result::unwrap()` on an `Err` value: SystemTimeError(1s)' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(1s)' panicked at '', called `Result::unwrap()` on an `Err` value: SystemTimeError(1s)', called `Result::unwrap()` on an `Err` value: SystemTimeError(1s)src/fs/file.rs', src/fs/file.rs', :src/fs/file.rs:src/fs/file.rs332:332::332:33271:71:
71
71

stack backtrace:
<unnamed>' panicked at 'thread 'called `Result::unwrap()` on an `Err` value: SystemTimeError(1s)<unnamed><unnamed>' panicked at '<unnamed>called`Result::unwrap()` on an `Err` value: SystemTimeError(1s)' panicked at '', ', called `Result::unwrap()` on an `Err` value: SystemTimeError(1s)'panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(1s)src/fs/file.rs', src/fs/file.rs:src/fs/file.rs332::', :33271src/fs/file.rs332:
::7133271:
71

0:     0x557b892f6ea8 - <unknown>
1:     0x557b8928126c - <unknown>
2:     0x557b892f6556 - <unknown>
3:     0x557b892f5f5a - <unknown>
4:     0x557b892f57e1 - <unknown>
5:     0x557b892f546a - <unknown>
6:     0x557b8927f920 - <unknown>
7:     0x557b892821c2 - <unknown>
8:     0x557b892b1128 - <unknown>
9:     0x557b892b36ec - <unknown>
10:     0x557b892b513d - <unknown>
11:     0x557b89297df4 - <unknown>
12:     0x557b892efbeb - <unknown>
13:     0x557b892ef40d - <unknown>
14:     0x557b89300725 - <unknown>
15:     0x7fa47f46a3e9 - start_thread
16:     0x7fa47f37e293 - __GI___clone
17:                0x0 - <unknown>
stack backtrace:
0: [1]    93997 abort (core dumped)  RUST_BACKTRACE=full exa -la

~/Pictures/Wallpaper
➜ ls -v
exa v0.9.0
```

**Comment by danwolb02** [comment]
According to the Rust documentation, this happens "whenever the second system time represents a point later in time than the self of the method call." ([Rust Documentation; SystemTimeError](https://doc.rust-lang.org/std/time/struct.SystemTimeError.html)). This means the modified-time metadata of a file lays in the future ("The clocks have gone backwards", should not be possible). I created a pull request: de2acc5

**Comment by ogham** [comment]
Yeah, this is caused by exa not handling the timestamp. The fix for this has been merged (#670) and will go out with the next release.
